### DISCUSSION TITLE
Multi select/add to lists

### DISCUSSION THREAD

**Discussion by ronbruins** [karakeep-app/karakeep#116] [Ideas]
I ran into this app and it is exactly what I am looking for! I just imported all my bookmarks, and then found that I would like to be able to multiselect items and bulk add them to a list. is this on the roadmap?

**Comment by MohamedBassem** [comment]
Hi, this is indeed in the roadmap, and tracked in https://github.com/MohamedBassem/hoarder-app/issues/84.
